app/assets/stylesheets/top_bar_styling.scss

```diff
@@ -0,0 +1,20 @@
+// #background {
+//  background: url(/assets/Boston_Skyline_Night_BW.png) no-repeat fixed center;
+//  background-size: contain;
+//  background-repeat: no-repeat;
+//  width: 100%;
+//  height: 100%;
+//  padding-top: 59.05%;
+//}
+
+$box-bg: #600000;
+$box-border: #400000;
+$font-color: #808080;
+
+.large-3 {
```

Comment:
I would use a different class name for this piece of styling. `.large-3` is something Foundation uses a lot, and you're liable to want to use elsewhere in your site outside the topbar. However, chances are you don't want this styling *everywhere* you could possibly want a column of this width.  Instead, add a new class name to wherever you want that styling, and use that name here instead of `large-3`